signal.go

```diff
@@ -0,0 +1,65 @@
+package signal
+
+import "math"
+
+type (
+	// Discrete signal
+	Discrete []float64
+
+	// Continuous does not exists inside the computer
+	// but let the bullshit continue.
+	Continuous func(x float64) float64
```

Comment:
Since it is represented by a function it may be infinite right ? =P

Reply:
It could be infinite in length, but not continuous. Independently of the function used, it will have discrete values because of the limits of float64.
In the computer, between two real numbers there's not infinite numbers. :D

Reply:
Hmm, maybe not infinite in length also kkkk

Reply:
yeah I meant length =) I saw a infinite set represented with functions before

Reply:
how that's possible?

Reply:
I dont remember the details anymore, it has been a long time ago =(
In the end you have to store some data somewhere, even if the set is sparse...soh infinite is a strong word for anything on a computer =P